graphql_compiler/compiler/expressions.py

```diff
@@ -305,6 +319,18 @@ def to_gremlin(self):
         else:
             return u'{}.{}'.format(local_object_name, self.field_name)
 
+    def to_sql(self, aliases, current_alias):
+        """Return a sqlalchemy Column picked from the current_alias."""
+        self.validate()
+
+        if isinstance(self.field_type, GraphQLList):
+            raise NotImplementedError(u'We dont support lists yet')
+
+        if '@' in self.field_name:
+            raise NotImplementedError(u'We dont support __typename yet')
```

Comment:
nit: `dont` -> `don't` and let's mention which field and type were the problematic ones.